File: openmrs-client/src/main/java/org/openmrs/mobile/models/FormData.java
package org.openmrs.mobile.models;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class FormData {

    @SerializedName("name")
    @Expose
    private String name;
    @SerializedName("valueReference")
    @Expose
    private String valueReference;
    @SerializedName("dataType")
    @Expose
    private String dataType;

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getValueReference() {
        return valueReference;
    }

    public void setValueReference(String valueReference) {
        this.valueReference = valueReference;
    }

    public String getDataType() {
        return dataType;
    }

    public void setDataType(String dataType) {
        this.dataType = dataType;
    }

}

File: openmrs-client/src/main/java/org/openmrs/mobile/models/FormData.kt
package org.openmrs.mobile.models

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class FormData {

    @SerializedName("name")
    @Expose
    var name: String? = null

    @SerializedName("valueReference")
    @Expose
    var valueReference: String? = null

    @SerializedName("dataType")
    @Expose
    var dataType: String? = null

}